eng/tools/ToolDescriptionEvaluator/Program.cs

```diff
@@ -2,7 +2,6 @@
 // Licensed under the MIT License.
```

Comment:
I have a tool that executes azmcp and performs some analysis similar to this ToolDescriptionEvaluator. Is it possible to pull out some common functionality from this into a class like MicrosoftMcpRepository with some methods like: GetRepoRoot(), GetToolDirectory(), GetCLIArtifact(), GetTools(), GetWorkDirectory() (for the throwaway stuff)?